src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTask.java

```diff
@@ -49,11 +50,10 @@ public void deleteMessagesInEnvelopesDlq() throws ServiceBusException, Interrupt
             while (message != null) {
                 if (canBeDeleted(message)) {
                     logMessage(message);
-                    messageReceiver.complete(message.getLockToken());
+                    messageReceiver.completeAsync(message.getLockToken());
                 }
                 message = messageReceiver.receive();
             }
-
         } catch (ConnectionException e) {
```

Comment:
Not a part of this PR, but we should catch and log all exceptions.

Reply:
@lgonczar we are catching the `ServiceBusException` and `InterruptedException` in `DlqReceiverProvider` class (from where we are getting the message reciever for dlq)

Reply:
It's still worth having some generic error logging, in case a runtime exception is thrown. Not necessarily in this PR.